### ISSUE TITLE
Undefined array key "content-type"

### ISSUE THREAD

**Issue by flaviencrochard** [simplepie/simplepie#885]
Hello,
When a feed has no results :
```
* Uses proxy env variable no_proxy == 'localhost,127.0.0.0/8,::1'
*   Trying 172.16.1.129:8011...
* Connected to 172.16.1.129 (172.16.1.129) port 8011 (#0)
> GET /merlin-service/rest/api/product/185070/restricted-prescriptions HTTP/1.1
> Host: 172.16.1.129:8011
> User-Agent: curl/7.88.1
> Accept: */*
> 
< HTTP/1.1 204 
< ETag: "1724277600000"
< Expires: -1
< Cache-Control: no-cache, no-transform, must-revalidate
< Access-Control-Allow-Origin: *
< Access-Control-Allow-Methods: GET, POST, OPTIONS
< Date: Mon, 23 Sep 2024 07:33:35 GMT
< 
* Connection #0 to host 172.16.1.129 left intact
```
If the Content-Type header is missing, it throws an ErrorException and the execution stops :
![image](https://github.com/user-attachments/assets/0ddc5c65-06ce-440e-a82e-864eb1c8635f)

Could it be repaired to get another exception or empty collection ?

Thanks for your help !

**Comment by jtojnar** [comment]
Thanks for reporting, this was probably fixed by https://github.com/simplepie/simplepie/commit/8fc95caf3c0f944191ca77fa57c70db034f88b8f.

**Comment by flaviencrochard** [comment]
Yes ! Do you know when this will be available in next release ? :)

**Comment by jtojnar** [comment]
Actually, we should probably fix it in 1.8 as well: https://github.com/simplepie/simplepie/blob/1.8.0/src/SimplePie.php#L1836

**Comment by jtojnar** [comment]
The fix has been merged. Hopefully, 1.8.1 will come out soon.
